mlflow/metrics/metric_definitions.py

```diff
@@ -16,20 +16,37 @@ def standard_aggregations(scores):
     }
 
 
-def _validate_text_data(data, metric_name, column_name):
-    """Validates that the data is text and is non-empty"""
-    if len(data) == 0:
-        return False
+def _is_text_data_invalid(data, metric_name, column_name):
+    """Validates that data list a list of strs and is non-empty"""
+    if data is None or len(data) == 0:
+        return True
```

Comment:
The comment says "Validates that data list a list of strs and is non-empty". Should it be `False` here?